### PR TITLE
Converter for LogicalOr and LogicalAnd operations

### PR DESCRIPTION
**Pull request by drivanov**
Implement  converters for LogicalOr and LogicalAnd operations.

### CODE REVIEW

File: tensorflow/compiler/tf2tensorrt/convert/ops/binary_ops.cc [hunk 1]
```diff
@@ -37,11 +37,21 @@ const binaryOperationMap *BinaryOperationMap() {
...
+  });
+  return m;
+}
+
 class ConvertBinaryImpl {
  protected:
   ConvertBinaryImpl(const binaryOperationMap *pOperMap) : pOperMap_(pOperMap) {}
 
-  Status ImplValidate(const OpConverterParams &params) {
+  Status ImplValidate(const OpConverterParams &params,
```
Comment (bixia1) [inline tensorflow/compiler/tf2tensorrt/convert/ops/binary_ops.cc:52]: We use XImpl in the existing code. Let's  do this
s/ImplValidate/ValidateImpl/
s/ImplConvert/ConvertImpl/

File: tensorflow/compiler/tf2tensorrt/convert/ops/binary_ops.cc [hunk 2]
```diff
@@ -57,6 +67,22 @@ class ConvertBinaryImpl {
...
+   if (!not_supported_ops.empty() && params.use_implicit_batch) {
+      const auto& end = not_supported_ops.end();
+      if (std::find(not_supported_ops.begin(), end, op) != end) {
+        return errors::Unimplemented(
+            "Binary op: '", op, "' is not supported in implicit batch mode");
+      }
+    }
+
+    if (both_tensors) {
+      // Need to check if both inputs should be tensors.
```
Comment (bixia1) [inline tensorflow/compiler/tf2tensorrt/convert/ops/binary_ops.cc:79]: This comment is the same as the line before, it is better without this comment.

**Review by bixia1 — COMMENTED** [review]
LGTM, just two minor requests. Thanks!

File: tensorflow/compiler/tf2tensorrt/convert/ops/binary_ops.cc (hunk 2 above)
Comment (drivanov) [reply tensorflow/compiler/tf2tensorrt/convert/ops/binary_ops.cc]: Done

File: tensorflow/compiler/tf2tensorrt/convert/ops/binary_ops.cc (hunk 1 above)
Comment (drivanov) [reply tensorflow/compiler/tf2tensorrt/convert/ops/binary_ops.cc]: Done

**Comment by DEKHTIARJonathan** [comment]
@bixia1 please review

**Review by bixia1 — APPROVED** [review]

**Review by bixia1 — CHANGES_REQUESTED** [review]
this PR also has build error, similar to PR [55428](https://www.google.com/url?sa=D&q=https%3A%2F%2Fgithub.com%2Ftensorflow%2Ftensorflow%2Fpull%2F55428).

Also, need to squash the commits into one.

**Comment by drivanov** [comment]
This PR has been replaced by [PR#55548](https://github.com/tensorflow/tensorflow/pull/55548)

**Comment by drivanov** [comment]
This PR has been replaced by [PR#55548](https://github.com/tensorflow/tensorflow/pull/55548)

**Pull request closed without merging**